dozer-sql/src/pipeline/product/set.rs

```diff
@@ -45,14 +48,69 @@ impl SetOperation {
         &self,
         action: SetAction,
```

Comment:
not sure we need this `action`, since we're returning the same value

Reply:
we are using that actions in the `execute_union()` function -> https://github.com/getdozer/dozer/pull/961/files#diff-d84fd3c1d6ca93290e4336fcdfad00967e63bf4abc47280583974d5af8815d56R57